src/sonic-host-services/scripts/hostcfgd

```diff
@@ -109,6 +130,22 @@ class Feature(object):
             raise ValueError('Invalid state rendered for feature {}: {}'.format(self.name, target_state))
         return target_state
 
+    def __safe_eval_bool(self, val):
```

Comment:
Default being False is specific to this case and also this isn't used anywhere else in the code, so i don't think it's required? Let me know if you think otherwise.

Reply:
I see `self` is not used in this function, so it is a class level static method, not an instance method. And you give `__` prefix to hide it, but it is seem useful as a public method.